nss-day-cohort-51/holiday-road-grand-canyon-gliders
# bw-preview

# Description

Added functionality to how the dropdown returns values. Also populating the Trip Preview section with the names of the selected values depending on selection. 

## Type of change


# Testing Instructions

To test functionality open dev tools upon loading and begin selecting a state, park, eatery, and bizarrerie and notice how the console will log changes to the local array. Also the Trip Preview section in the DOM should update with each change to park, eatery, and bizarrerie.


# Checklist:
